modules/tinymce/src/plugins/table/test/ts/browser/settings/TableToolbarTest.ts

```diff
@@ -50,4 +52,55 @@ describe('browser.tinymce.plugins.table.TableToolbarTest', () => {
     await TinyUiActions.pWaitForUi(editor, '.tox-pop .tox-tbtn[title="Paste row before"]:not(.tox-tbtn--disabled)');
     McEditor.remove(editor);
   });
+
+  context('Ensure table buttons are enabled when appropriate', () => {
```

Comment:
This isn't a great place for this... this test is checking the context toolbar rendering logic and the `table_toolbar` setting. Your new tests don't use any of that, so please move them to a separate file. You also don't need to be initializing an editor per test, as you aren't changing any settings.